app/src/main/kotlin/com/wire/android/model/DialogState.kt

```diff
@@ -0,0 +1,6 @@
+package com.wire.android.model
+
+sealed class DialogState<out T : Any> {
```

Comment:
Can't the same be achieved by simply making it nullable? (null means hidden, non-null means visible)

Reply:
But does it tells you when you are in view model what is it purpose?